### PR TITLE
dyncall: update to 1.3

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update to 1.3. Changed `cmake` PortGroup to 1.1.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.6 PPC (10A190)
Xcode 3.2

macOS 10.5.8
Xcode 3.1.4

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by barracuda156** [comment]
@reneeotten Could you say how I can tag the port maintainer when the bot doesn’t do that? I don’t know his Github ID.

**Comment by reneeotten** [comment]
> @reneeotten Could you say how I can tag the port maintainer when the bot doesn’t do that? I don’t know his Github ID.

You cannot if you don't know the GitHub handle; you can use the e-mail address listed in the Portfile to notify the maintainer about this PR.

File: devel/dyncall/Portfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
 
 name                dyncall
-version             1.2
+version             1.3
 categories          devel
 platforms           darwin
```
Comment (reneeotten) [inline devel/dyncall/Portfile:9]: you could remove this line

File: devel/dyncall/Portfile [hunk 2]
```diff
@@ -23,12 +23,15 @@ long_description    DynCall is a suite of three libraries (dyncall, dynload \
 homepage            http://dyncall.org/
```
Comment (reneeotten) [inline devel/dyncall/Portfile:23]: please use `https`

**Comment by barracuda156** [comment]
@reneeotten Thank you! Will fix these now.

**Comment by barracuda156** [comment]
> > > @reneeotten Could you say how I can tag the port maintainer when the bot doesn’t do that? I don’t know his Github ID.
> > 
> > You cannot if you don't know the GitHub handle; you can use the e-mail address listed in the Portfile to notify the maintainer about this PR.
> 
> @barracuda156 did you get a response from the maintainer on your e-mail notifying him about this PR?

I am sorry, this part went forgotten. I will send an e-mail, the website should have a contact. Anyway, I was planning to add `ppc64` support and ask for advice in that regard.

**Comment by barracuda156** [comment]
@reneeotten Got reply: “Thanks for using dyncall and letting us know about the pull request. 
That fix to the callback_suite makefile is already done in the dyncall sources and will be part of the upcoming 1.4 release. Sorry that this error was in the 1.3 release, this was obviously not intended”.

**Pull request merged**